.github/pull_request_template.md

```diff
@@ -0,0 +1,8 @@
+## [HELPS-XXXX](https://crosschx.atlassian.net/browse/HELPS-XXXX)
+
+### Changes made:
+
+- Bullet
```

Comment:
Why bullet point here? 0.0

Reply:
This is a PR template, the bullet points are there to use when you open a PR :)